### DISCUSSION TITLE
Nuclear war scenario for the Lojban language

### DISCUSSION THREAD

**Discussion by lagleki** [lojban/casnu#13] [General]
How can Lojban materials be saved offline?
Scenarios:
- electronic devices continue to exist:
  - offline apps and books
- electronic devices are unavailable:
  - tutorials, texts inscribed on rocks etc.

**Comment by GJ-u** [comment]
the cold war is over, gleki. no need to worry about this too urgently.

**Reply by arran4** [reply]
I think it's mean to be figurative. There are other things which can happen: https://fallofcivilizationspodcast.com/ Plus some amongst us do the service of archiving what they can.

**Comment by lagleki** [comment]
[la sutysisku](https://github.com/La-Lojban/sutysisku) dictionary app works offline in modern browsers.

**Comment by lagleki** [comment]
The CLL can be saved as a single pdf file.

**Comment by ghost** [comment]
`wget | pandoc` worked well.
